image-tiff/src/main.rs

```diff
@@ -0,0 +1,18 @@
+#[macro_use]
+extern crate honggfuzz;
+extern crate tiff;
+
+// to run this fuzzer use honggfuzz to run image-tiff
+use std::io::Cursor;
+
+fn main() {
+    loop {
+        fuzz!(|data: &[u8]| {
```

Comment:
Shoudn't this value be used in the test body?

Reply:
done!! now the cursor is created from the data